realm-jni/src/io_realm_internal_tableview.cpp

```diff
@@ -860,6 +860,40 @@ JNIEXPORT void JNICALL Java_io_realm_internal_TableView_nativeSort(
     } CATCH_STD()
 }
 
+JNIEXPORT void JNICALL Java_io_realm_internal_TableView_nativeSortMulti(
+  JNIEnv* env, jobject, jlong nativeViewPtr, jlongArray columnIndices, jboolean ascending)
+{
+    try {
+        if (!VIEW_VALID_AND_IN_SYNC(env, nativeViewPtr))
+            return;
+        std::vector<size_t> indices;
+        std::vector<bool> ascendings;
+        jsize arr_len = env->GetArrayLength(columnIndices);
+        jlong *arr = env->GetLongArrayElements(columnIndices, NULL);
+        for (int i=0; i<arr_len; ++i) {
+            if (!COL_INDEX_VALID(env, TV(nativeViewPtr), arr[i]))
+                return;
+            int colType = TV(nativeViewPtr)->get_column_type( S(arr[i]) );
+            switch (colType) {
+                case type_Bool:
```

Comment:
These are core's types, and `type_Int` is the general integer column type. I will check if core supports sorting binary (byte) data.